src/Mobs/Monster.cpp

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Code convention, for some reason global code search in VS needed a blank line at the start of the file. Ask @madmaxoft for more info.